test/ci/azure/ubuntu18.yml

```diff
@@ -24,50 +29,55 @@
  
    - script: |
       pip3 install -r requirements.txt
-      pip3 install -U yapf==0.26.0 pytest 
+      pip3 install -U pytest 
      displayName: "Setup" 
+
+   - script: |
+      wget https://registrationcenter-download.intel.com/akdlm/irc_nas/18617/l_openvino_toolkit_p_2022.1.0.643_offline.sh
+      chmod +x l_openvino_toolkit_p_2022.1.0.643_offline.sh
+      ./l_openvino_toolkit_p_2022.1.0.643_offline.sh -a -s --eula accept --install-dir $(Pipeline.Workspace)/openvino/
+     displayName: "Install OpenVINO"
     
    - script: |
       git submodule init
       git submodule update
-      python3 build_ovtf.py --use_tensorflow_from_location=$(TF_LOCATION) --use_openvino_from_location=$(OV_LOCATION) --openvino_version=2022.1.0 --cxx11_abi_version=1
+      python3 build_ovtf.py --use_openvino_from_location=$(OV_LOCATION) --openvino_version=2022.1.0 --cxx11_abi_version=1
      workingDirectory: $(WORK_DIR) 
      displayName: "Build"
      
    - script: |
       source build_cmake/venv-tf-py3/bin/activate
-      pip3 install -U build_cmake/artifacts/tensorflow/tensorflow-*.whl
       pip3 install --no-deps -U build_cmake/artifacts/openvino_tensorflow*.whl
      workingDirectory: $(WORK_DIR) 
      displayName: "Install"
     
-   - script: |
-      source $(WORK_DIR)/build_cmake/venv-tf-py3/bin/activate
-      ./maint/check-code-format.sh
-     workingDirectory: $(WORK_DIR) 
-     displayName: "Code format check"
+  #  - script: |
```

Comment:
We can keep these blocks as we may need them later.

Reply:
Sure, included code-format-check for ubuntu18.